EMML/1001-2000/EMML1292.xml

```diff
@@ -48,6 +48,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://ra
                   <collection>Project EAP286:
                     Digitising and conserving Ethiopian manuscripts at the Institute of Ethiopian Studies</collection>
                   <idno>EAP286-1-1-6</idno>
+                </altIdentifier>
```

Comment:
this is wrong. The identifier is IES 00019 as already given. MS EMIP 00019 is a very different one, `EMIP00019`
please remove this altIdentifier or even better retract the commit from this branch

Reply:
Can I use "Revert changes in commit" for this, in GitHub?